docs/rest-api/source/includes/wp-api-v3/order.md

```diff
@@ -0,0 +1,141 @@
+# Order
+
+The order API allows you to perform operations such as capture a terminal payment, create an intent, create a customer, etc. for a particular order.
+
+## Create a terminal intent
+
+Create a new in-person payment intent for the given order ID without [confirming](https://stripe.com/docs/api/payment_intents/confirm) it.
```

Comment:
If we plan to make this public then we shouldn't link to Stripe

Reply:
Fixed in 5ecd764